server/sonar-web/src/main/js/apps/quality-profiles/home/UsefulInformation.js

```diff
@@ -17,27 +17,19 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import Backbone from 'backbone';
-import Profile from './profile';
+import React from 'react';
 
-export default Backbone.Collection.extend({
-  model: Profile,
-  url: window.baseUrl + '/api/qualityprofiles/search',
-  comparator: 'key',
+export default class UsefulInformation extends React.Component {
+  render () {
+    return null;
 
-  parse (r) {
-    return r.profiles;
-  },
-
-  updateForLanguage (language) {
-    this.fetch({
-      data: {
-        language
-      },
-      merge: true,
-      reset: false,
-      remove: false
-    });
+    return (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this code after the "return" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AUnreachableCode)